build/target-repository/docs/static_reflection_and_autoload.md

```diff
@@ -90,3 +90,21 @@ In this case you may want to try one of the following solutions:
         //          as much as you can or you will slow down each run
     ]);
 ```
+
+Other solution is by register the path of the class to composer.json's `"files"` config, eg:
+
+```javascript
+    "autoload-dev": {
+        "files": [
+            "vendor/acme/my-custom-dependency/src/Your/Own/Namespace/TheAffectedClass.php"
+        ]
+    }
+```
+
+After that, run:
+
+```bash
+composer update
```

Comment:
Composer would also update `compoesr.lock` and change dependencies.

Here the `composer dump-autoload`  would be enough

```suggestion
composer dump-autoload
```

Reply:
updated https://github.com/rectorphp/rector-src/pull/1472/commits/66f744de903a273c76d4b6128ba71925b19bedb9